src/adapters/bidderFactory.js

```diff
@@ -296,6 +296,12 @@ export function newBidder(spec) {
  * @param onCompletion {function()} invoked once when all bid requests have been processed
  */
 export const processBidderRequests = hook('sync', function (spec, bids, bidderRequest, ajax, wrapCallback, {onRequest, onResponse, onError, onBid, onCompletion}) {
+  let updatedUserAsEids = config.getConfig('dcUsersAsEids');
```

Comment:
This looks like it's using `config` as a global variable, plus it's adding dead code for those who are not using the data controller module.

Why not do this directly from the hook added by the module? 
Also, IMO the `makeBidRequests` hook is a better place for this - I don't know if it makes a difference now, but my instinct is that the earlier they have this data the better.

Reply:
@dgirardi  I have update this module to be self contained.